orbit/cmd/bare/README.md

```diff
@@ -0,0 +1,51 @@
+# Fleet osquery extensions without fleetd
+
+If you are interested in getting some of the `fleetd` tables but cannot run `fleetd` natively then its possible
+to utilize this "bare" extension with standalone `osqueryd`.
+
+### Building the extension
+
+First run (note `.ext` is required for osquery):
+```shell
+go build -o bare.ext bare.go
```

Comment:
Let's add an entry in the main `Makefile`:
```sh
bare:
  go build -o bare.ext ./orbit/cmd/bare
```
So that users can just do `make bare`.

Was the name `bare` agreed upon? :) I like it.

Reply:
No I was hoping we'd sort it out here haha. 

+1 for Makefile

Reply:
If it doesn't use CGO, then we could use GOOS/GOARCH to allow cross compiling it.

Reply:
I think for now this would work, and if we feel like we need more we can iterate (distributing artifacts via github actions, etc.)